sdk/azidentity/cache/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.3.0
 	github.com/Azure/azure-sdk-for-go/sdk/internal v1.3.0
-	github.com/AzureAD/microsoft-authentication-extensions-for-go/cache v0.1.0
+	github.com/AzureAD/microsoft-authentication-extensions-for-go/cache v0.1.1-0.20230928163952-361da6dd79f8
```

Comment:
When will `v0.1.1` be released?

Reply:
I don't know yet, but I won't ship this feature before that release.